examples/basic/text/script.js

```diff
@@ -0,0 +1,30 @@
+import * as UTILS from '../../globals';
+
+const controlsModule = new WHS.OrbitControlsModule();
+
+const app = new WHS.App([
+  ...UTILS.appModules({
+    position: new THREE.Vector3(0, 10, 200)
+  }),
+
+  controlsModule
+]);
+
+const text = new WHS.Text({
+  text: 'Gentilis Bold Text',
+
+  parameters: {
+      font: `${process.assetsPath}/fonts/gentilis_bold.typeface.json`,
```

Comment:
Can we move it to:

```js
const text = new WHS.Text({
  text: 'Gentilis Bold Text',
  url: '/path/to/font.typeface.json'
});
```

And if dev wants to input a ready font:

```js
const text = new WHS.Text({
  text: 'Gentilis Bold Text',
  font: myFont
});
```

So the algorythm will use `url` if it is provided, if not - then check for `font`.